go/vt/sqlparser/parse_test.go

```diff
@@ -884,9 +884,6 @@ var (
 		input: "set @variable = 42",
 	}, {
 		input: "set @period.variable = 42",
-	}, {
-		input:  "alter ignore table a add foo",
-		output: "alter table a",
 	}, {
```

Comment:
why is this test case removed?

Reply:
`ALTER IGNORE` is an invalid syntax starting MySQL `5.7`. It was last valid on `5.6`. Since we only support `5.7` and above, we should not allow this syntax,